homeassistant/components/thermostat/zwave.py

```diff
@@ -109,6 +134,24 @@ def target_temperature(self):
     def set_temperature(self, temperature):
         """Set new target temperature."""
         # set point
+        if self._zxt_120 == 1:
```

Comment:
In that case, maybe it is better to create a general hvac component so more devices can be implemented later on.  If that is a better way, I will have a go at it.